crates/pop-cli/src/deployment_api.rs

```diff
@@ -285,7 +285,7 @@ mod tests {
 			json!({
 				"name": "Development",
 				"properties": {
-					"basedOn": "standard",
+					"basedOn": "r0gue-io/standard",
```

Comment:
```suggestion
					"basedOn": "r0gue-io/base-parachain",
```
As I remember from the feedback, we also want that user can easily copy this string and paste on google to find the repo: 

<img width="705" alt="Screenshot 2025-03-25 at 13 33 49" src="https://github.com/user-attachments/assets/fff2819d-7cae-4dbc-83f5-64031569736b" />

Reply:
You are right: https://github.com/r0gue-io/pop-cli/pull/483/commits/f2b0338f5ef9ac54176b195c137486d2d843a764